tests/test_check_processing_time.py

```diff
@@ -0,0 +1,54 @@
+import check_processing_time
+
+
+def test_single_attempt():
+    attempts = [{'Container': {}, 'StartedAt': 1644423555778, 'StatusReason': '', 'StoppedAt': 1644425055200}]
+    result = check_processing_time.get_time_from_attempts(attempts)
+    assert result == 1644425055200 - 1644423555778
```

Comment:
Test coverage looks good. In general, I prefer a test's final `assert` to compare against a constant to minimize the amount of thinking a reader has to do. Maybe one test should include real-ish timestamps, but I suspect most of them can be simplified to something like:
```suggestion
    attempts = [{'Container': {}, 'StartedAt': 11, 'StatusReason': '', 'StoppedAt': 13}]
    result = check_processing_time.get_time_from_attempts(attempts)
    assert result == 2
```